autopush/db.py

```diff
@@ -266,6 +268,12 @@ def all_channels(self, uaid):
     def store_message(self, uaid, channel_id, data, headers, message_id, ttl):
         """Stores a message in the message table for the given uaid/channel with
         the message id"""
+        if data:
+            # Store Base64-encoded payloads as binary fields.
+            if not isinstance(data, bytes):
```

Comment:
Under what conditions does this happen? Is it intentional that we'd pass it possibly bytes, but not?